com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfJoints/UrdfJoint.cs

```diff
@@ -292,12 +295,6 @@ public void GenerateUniqueJointName()
             jointName = transform.parent.name + "_" + transform.name + "_joint";
         }
 
-        protected static Joint.Axis GetAxisData(Vector3 axis)
-        {
-            double[] rosAxis = axis.ToRoundedDoubleArray();
-            return new Joint.Axis(rosAxis);
-        }
```

Comment:
Why wouldn't we keep this in base class still, but specialise with unity joint? Seems all that the derived classes do is to use the same implementation basically (with the exception of one)

Reply:
It is different for planar.